lighting/models/product_source_line.py

```diff
@@ -380,6 +440,64 @@ def _compute_wattage_display(self):
         string="Source",
     )
 
+    invisible_fields = fields.Json(
+        compute="_compute_invisible_fields",
+    )
+
+    def _get_all_dependent_fields(self):
+        self.ensure_one()
+        all_fields = set()
+        for fields_g in self.FIELD_GROUP_MAP().values():
+            all_fields |= set(fields_g)
+        return all_fields
+
+    def _get_irrelevant_dependent_fields(self, values=None):
```

Comment:
To be like the others?
```suggestion
    def  _get_dependant_irrelevant_fields(self, values=None):
```